ajax/libs/design-system/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "design-system",
+  "filename": "styles/salesforce-lightning-design-system.min.css",
+  "description": "Salesforce Lightning Design System",
+  "version": "2.0.2",
+  "author": "Salesforce",
+  "license": "BSD-2-Clause",
```

Comment:
The licenses are BSD-3-Clause, CC-BY-ND-4.0.
Ref: https://github.com/salesforce-ux/design-system#licenses.